src/aggregations/primitive_pairs.py

```diff
@@ -12,8 +12,14 @@
 
 
 class ScoreDiff(NamedTuple):
+    """
+    `metric_score_diff` is calculated as the the score of `run_a` (in the
+    containing PipelineRunPairDiffEntry) minus the score of `run_b`. I.e.,
+    a positive value indicates that `run_a` had a higher score, while a
+    negative value means `run_b` had a higher score.
+    """
```

Comment:
Beautiful documentation.

Reply:
It looks like down below on line 181 the actual computation is `run_b_score.value - run_a_score.value`.